Source/core/SocketPort.cpp

```diff
@@ -891,8 +891,9 @@ namespace Core {
             }
 
             if (l_Size == 0) {
+                uint32_t l_Result = __ERRORRESULT__;
```

Comment:
This line is now no longer needed, so guess we can remove it as well.

Reply:
Removed it